runtime/help/colors.md

```diff
@@ -106,7 +106,7 @@ be found
 Custom colorschemes should be placed in the `~/.config/micro/colorschemes`
 directory.
 
-A number of custom directives are placed in a `.micro` file. Colorschemes are 
+A number of custom directives are placed in a `.micro` file. Colorschemes are
```

Comment:
I suggest to remove these trailing white space changes, since you need to rebase your PR anyway. It's preferred to do only the changes necessary for the topic tracked within the PR.